ember-a11y/2016-04/april-27.md

```diff
@@ -0,0 +1,89 @@
+# ember-a11y Meeting Notes - 2016-04-27
+
+## Attendees
+
+- Nathan Hammond
+- Melanie Sumner
+- Jamie White
+- David Peter
+- Suz Hinton
+- Ben Holmes
+- Will Hastings
+- Amanda - CART
+
+## Actions
+
+- [ ] Everybody - Get updates into Jamie's blog post.
+- [ ] Nathan - Divide and assign a11y tools for review.
+- [ ] Nathan/Trent - Migrate ember-a11y-testing.
+- [ ] Rob - Find an a11y presenter for Ember ATX.
+- [ ] Nathan - Introduce Suz to Luke Melia re: EmberNYC
+- [ ] Rob - Shop Talk Show - Dave Rupert, Chris Coyier (possibly in Austin, introduction to Rob?)
+- [ ] Jamie - Work with Leonie to familiarize her with our efforts and code.
+- [ ] Nathan - Fix a11y-demo-app publish script.
+
+## Discussion Notes
+
+### Introductions
+
+- Will Hastings!
+- David Peter!
+
+### Speaking
+
+- We want to make sure that we're always out being visible in the community.
+- GEMConf - Nathan is speaking. Lightning talk.
+- EmberNYC - Get Suz to speak.
+- Ember ATX - Rob to find a presenter.
+- Shop Talk Show - Get Rob to speak?
+- EmberCamp London - Closing Keynote is Leonie Watson.
+
+### Relationships
+
+- Seattle, looking for somebdoy to work with Marcy Sutton.
+- NYC, connect Suz and Luke.
+
+### Outreach
+
+- Jamie's Blog Post has been moved into Canvas for editing.
+
+### Addon Community
+
+- @atomicc is working on making liquid-fire accessible.
+- ember-a11y-testing as it exists will be deprecated.
+- ember-axe will be renamed to ember-a11y-testing.
+- We will gain control over these as part of the ember-a11y org soon.
+
+### Logo
+
+- Mel created us a logo!
+- We'll use it until we're an official subteam and get the script version.
+
+### Demo App
+
+- Basic structure online [https://ember-a11y.github.io/a11y-demo-app/](https://ember-a11y.github.io/a11y-demo-app/)
+- Working through list in [design doc](https://usecanvas.com/ember-a11y/a11y-demo-app/4Tc35ruHgwbZCX1WRI3anp).
+- DOM structure is important, we can't just throw the components on a page.
+  - For now defer to patterns that Melanie is familiar with.
+  - Will do active research into markup after focus on fixing glaring Ember issues.
+  - Provide lots of examples.
+
+### Collection of a11y tools
+
+- We can totally steal some of approaches from other tools.
+  - https://github.com/prettydiff/a11y-tools
+  - https://twitter.com/burgessdryan/status/724384501326045184?s=09
+  - http://a11yproject.com/
+- We'll review each of these and regroup on things we should use or port to Ember.
```

Comment:
My action was to assign those, but that'll be for a report three weeks from now and will come separately from these notes. :)